lib/quickbooks/model/inventory_adjustment.rb

```diff
@@ -22,6 +22,7 @@ class InventoryAdjustment < BaseModel
       xml_accessor :customer_ref, from: 'CustomerRef', as: BaseReference
 
       xml_accessor :line_items, from: 'Line', as: [Line]
+      xml_accessor :shipping_adjustment, from: 'ShippingAdjustment'
```

Comment:
nit: can be changed to denoted its bool
```suggestion
      xml_accessor :shipping_adjustment?, from: 'ShippingAdjustment'
```